src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowirePassTest.php

```diff
@@ -413,6 +413,39 @@ public function testOptionalScalarArgsNotPassedIfLast()
             $definition->getArguments()
         );
     }
+
+    /**
+     * @dataProvider getCreateResourceTests
+     */
+    public function testCreateResourceForClass($className, $isEqual)
+    {
+        $startingResource = AutowirePass::createResourceForClass(
+            new \ReflectionClass(__NAMESPACE__.'\ClassForResource')
+        );
+        $newResource = AutowirePass::createResourceForClass(
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Symfony\Component\DependencyInjection\Compiler\AutowirePass' in method 'testCreateResourceForClass'.](https://www.codacy.com/app/git_9/symfony/file/2331919433/issues/source?bid=3207821&fileBranchId=3207866#l425)